packages/open-truss/src/signals/index.tsx

```diff
@@ -186,6 +186,14 @@ export const BooleanSignal = SignalType<boolean>(
   'boolean',
   z.boolean().default(false),
 )
+export const ObjectsSignal = SignalType<Array<Record<string, unknown>>>(
+  'object[]',
+  z.array(z.object({})).default([]),
+)
+export const ObjectSignal = SignalType<Record<string, unknown>>(
+  'object',
```

Comment:
Oh interesting. So is this for the case where you may just want to grab the entire blob from an endpoint instead of specific fields?

I think this is fine in principle, in practice I think we may want to encourage as much typing as possible when designing components where it makes sense. Some cases probably could benefit from something like `object`, but most cases it probably makes sense to use narrower types which will help us build out a library that is more linkable.

Reply:
Agreed 👍 

I pushed a commit updating the zod typing for this component and removing these exports.